README.md

```diff
@@ -1,2 +1,134 @@
-# OOP-school-library
-In this project, you will start building your school library app. Implement classes and objects in Ruby. Implement encapsulation and inheritance with Ruby. Run a program using the command line.
+<a name="readme-top"></a>
+
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
I added the **first version of my logo** on the readme file, sir. 

![fix-02](https://github.com/jlcarrascof/OOP-school-library/assets/86327867/55e8453e-2040-41b9-a741-3c29fcdd19b7)